ecommerce/data/model.py

```diff
@@ -1,9 +1,14 @@
-from sqlmodel import SQLModel, Field
-from ecommerce.User.model import Users
+from sqlmodel import SQLModel, Field, create_engine
+from user.model import Users
+
+engine = create_engine("postgresql://postgres:postgres@localhost/postgres")

```

Comment:
Duplicated engine instance, engine should be declared only once within whole app, as you did in first file above.